select.scss

```diff
@@ -0,0 +1,127 @@
+// Select
+// ===
+
+
+// Configurable Variables
+// ---
+
+// Whether to draw the default down-arrow icon.
+$select__draw-caret: true !default;
+
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'spline/dist/utilities/triangle';
+@import 'level/level';
+
+
+// Root
+// ---
+
+.c-select {
```

Comment:
Should this have a default white background? Right now, the test looks like this:

![screen shot 2015-05-29 at 11 18 35 am](https://cloud.githubusercontent.com/assets/734535/7889452/777d1e5c-05f4-11e5-8444-9d3afd33465c.png)